packages/phone-number-privacy/common/src/utils/input-validation.ts

```diff
@@ -1,28 +1,39 @@
 import { isValidAddress, trimLeading0x } from '@celo/utils/lib/address'
+import isBase64 from 'is-base64'
+import { isValidPhoneNumber } from 'libphonenumber-js'
+import {
+  GetBlindedMessageSigRequest,
+  GetContactMatchesRequest,
+  GetQuotaRequest,
+} from '../interfaces'
 import { REASONABLE_BODY_CHAR_LIMIT, REQUEST_EXPIRY_WINDOW_MS } from './constants'
 
 export function hasValidAccountParam(requestBody: any): boolean {
   return requestBody.account && isValidAddress(requestBody.account)
 }
 
-export function hasValidUserPhoneNumberParam(requestBody: any): boolean {
-  return !!requestBody.userPhoneNumber
+export function hasValidUserPhoneNumberParam(requestBody: GetContactMatchesRequest): boolean {
+  return !!requestBody.userPhoneNumber && isValidPhoneNumber(requestBody.userPhoneNumber)
 }
 
-export function hasValidContractPhoneNumbersParam(requestBody: any): boolean {
-  return requestBody.contactPhoneNumbers && Array.isArray(requestBody.contactPhoneNumbers)
+export function hasValidContactPhoneNumbersParam(requestBody: GetContactMatchesRequest): boolean {
+  return (
+    !!requestBody.contactPhoneNumbers &&
```

Comment:
Nit: I'm pretty sure these two lines are redundant? I may be missing something.

Reply:
True, we don't need to check the property exists if we're then immediately calling `Array.isArray`